nltk/app/collocations_app.py

```diff
@@ -324,23 +340,27 @@ def run(self):
                 words = self.model.CORPORA[self.name]()
                 from operator import itemgetter
                 text = [w for w in words if len(w) > 2]
-                fd = FreqDist(tuple(text[i:i+2]) for i in range(len(text)-1))
+                fd = FreqDist(tuple(text[i:i + 2])
+                              for i in range(len(text) - 1))
                 vocab = FreqDist(text)
-                scored = [((w1,w2), fd[(w1,w2)] ** 3 / (vocab[w1] * vocab[w2])) for w1, w2 in fd]
+                scored = [((w1, w2), fd[(w1, w2)] ** 3 /
+                           (vocab[w1] * vocab[w2])) for w1, w2 in fd]
                 scored.sort(key=itemgetter(1), reverse=True)
                 self.model.collocations = list(map(itemgetter(0), scored))
                 self.model.queue.put(CORPUS_LOADED_EVENT)
             except Exception as e:
                 print(e)
                 self.model.queue.put(ERROR_LOADING_CORPUS_EVENT)
 
-#def collocations():
+# def collocations():
```

Comment:
The reason for no space is that a commented-out code block can be easily commented-in.